scripts/random_volume_transformations.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
Is in in scripts instead of docs on purpose? You can add a separate dir called notebooks for non-documentation notebooks